<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>679</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>85</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>222</green>
        <blue>222</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>85</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>153</green>
        <blue>153</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>222</green>
        <blue>222</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Monitor Del Sistema</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="panel_principal" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAcerca_De">
    <property name="title">
     <string>Ace&amp;rca De</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLanzador_de_Aplicaciones">
    <property name="title">
     <string>&amp;Lanzador de Aplicaciones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMonitoreo">
    <property name="title">
     <string>&amp;Monitoreo</string>
    </property>
   </widget>
   <addaction name="menuMonitoreo"/>
   <addaction name="menuLanzador_de_Aplicaciones"/>
   <addaction name="menuAcerca_De"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
